presto-geospatial-toolkit/pom.xml

```diff
@@ -75,4 +75,19 @@
         </dependency>
 
     </dependencies>
+
```

Comment:
Why don't we move this change and the below changes to the `duplicate-finder-maven-plugin` config to the other commit?

Reply:
Done.